aikiblog/templates/aikiblog/techtren_list.html

```diff
@@ -0,0 +1,38 @@
+{% extends "base.html" %}
+{% block content %}
+	<a name="content"></a>
+	<p>Twoje techniki <b>{{ technique }}</b>:</p>
+	<table>
+	<tr>
+		<td id="table_techniques_date_cell"><b>Data</b></td>
+		<td id="table_techniques_stand_cell"><b>Postawa</b></td>
+		<td id="table_techniques_attack_cell"><b>Atak</b></td>
+		<td id="table_techniques_mistakes_cell"><b>Błędy</b></td>
+	</tr>
+
+{% for entry in techtren_list %}
+	<tr>
+		<td valign="top" id="table_techniques_date_cell">{{ entry.date|date:"d b y" }}</td>
+		<td valign="top" id="table_techniques_stand_cell">{{ entry.stand }}</td>
+		<td valign="top" id="table_techniques_attack_cell">{{ entry.attack }}</td>
+		<td valign="top" id="table_techniques_mistakes_cell">{{ entry.mistakes }}</td>
+		</tr>
+{% endfor  %}
+	</table>
+	<table>
+	<tr>
+		{% if is_paginated %}
+			{% if page_obj.has_previous %}
+				<td id="pagination_left"><a href="{% url "technique-list" technique.slug page_obj.previous_page_number %}#content">&lt;&lt;nowsze</a></td>
```

Comment:
Ok, to samo możesz zrobić dla class="pagination-left" i w cssach: .pagination-left {}
Wiem, że w przykładach w internetach jest, że po id, ale to brzydkie.